quickstarts/maven/hello-world/README.md

```diff
@@ -3,93 +3,54 @@ name: "Maven :: Hello World"
 description: |
   Demo project for getting started with Eclipse JKube.
   It just prints "Hello World" on command line and exits.
+author: Wayne Kirimi
```

Comment:
Not sure if it's okay to add author field here.

This description is rendered on Quickstarts Page [here](https://eclipse.dev/jkube/quickstarts/)